Fabric/src/main/java/com/blamejared/crafttweaker/CraftTweakerFabric.java

```diff
@@ -38,6 +40,16 @@ public void onInitialize() {
         
         Services.PLATFORM.registerCustomTags();
         
+        CraftTweakerEvents.REGISTER_CUSTOM_BEP_EVENT.register((loader, regFun) -> {
+            if(CraftTweakerConstants.MOD_ID.equals(loader)) {
+                // Fire the old event for compatibility: this is only required for the CrT loader
+                Services.EVENT.fireRegisterBEPEvent(new WrappingBracketParser(regFun));
+            }
```

Comment:
When reading I was confused about where the BEPs would be added here, until I figured out that they are implicitly added by the old event listener, maybe add a comment here to state that 😉